commands/urlib.go

```diff
@@ -269,11 +275,12 @@ func (u *URLib) handleURLibAdd(_ *discordgo.Session, ic *discordgo.InteractionCr
 
 	resp := fmt.Sprintf("Added `%s`.", ur)
 	u.Add(&UResource{
-		URL:      urp,
+		URL:      urp.String(),
 		Keywords: keywords,
 		Title:    title,
 		Added:    time.Now(),
-		Author:   *ic.Member.User,
+		Author:   *&ic.User.Username,
+		ID:       *&ic.User.ID,
```

Comment:
```suggestion
		Author:   ic.Member.User.Username,
		ID:       ic.Member.User.ID,
```
According to the dgo docs, `Member` is set when the command is invoked in a guild, and `User` in a DM.
We should probably use `Member`, do we want to support DMs at all?